app/src/main/res/layout-sw600dp/options_without_drawer_activity.xml

```diff
@@ -9,9 +9,15 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <include
-      android:id="@+id/options_activity_toolbar"
-      layout="@layout/toolbar" />
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/options_activity_toolbar"
```

Comment:
This is duplicating the ID from option_activity--I'm assuming that's an oversight from the original implementation. Please update it to be unique for this activity.

Reply:
Updating the id to be unique, triggers 8 tests to fail, as shown by the images. Could it be that they weren't oversights after all? @BenHenning  PTAL. 

![dup1](https://user-images.githubusercontent.com/31986629/197180188-4635387b-65a2-47ca-a5bf-92f94b402b53.PNG)

![dup2](https://user-images.githubusercontent.com/31986629/197180194-f490192a-53f4-4aaf-a921-941de9a007b4.PNG)

![dup3](https://user-images.githubusercontent.com/31986629/197180198-bd53ae1e-7611-4a87-8ba3-4e6e0764adf9.PNG)

![dup4](https://user-images.githubusercontent.com/31986629/197180203-ae4773c5-a826-4f0f-ad0d-d666276563f8.PNG)

![dup5](https://user-images.githubusercontent.com/31986629/197180212-93fa1a16-b15e-4ef7-a624-40368b5f1092.PNG)

![dup6](https://user-images.githubusercontent.com/31986629/197180214-719118d3-abac-45c1-8f71-8d711b4840c9.PNG)

Reply:
This means that those tests need to be changed @Ryggs. That being said, I don't mind keeping the status quo here since it's unrelated to your PR.